core/templates/pages/exploration-editor-page/editor-tab/services/responses.service.ts

```diff
@@ -237,28 +271,17 @@ angular.module('oppia').factory('ResponsesService', [
           callback(_answerGroupsMemento, _defaultOutcomeMemento);
         }
       },
-      getActiveAnswerGroupIndex: function() {
-        return _activeAnswerGroupIndex;
-      },
       changeActiveAnswerGroupIndex: function(newIndex) {
         // If the current group is being clicked on again, close it.
-        if (newIndex === _activeAnswerGroupIndex) {
```

Comment:
No, this should be reverted. As the comment suggests, this is required to close the answer group modal when it is clicked on again., which won't happen without this check.

Reply:
Done.